app/views/trainees/_filters.html.erb

```diff
@@ -0,0 +1,89 @@
+<form method="get" id="js-live-filter">
```

Comment:
Pulled from `index.rb` - no new code except for addition of `js` ids to use as selectors.